lms/djangoapps/teams/static/teams/js/spec/views/team_discussion_spec.js

```diff
@@ -18,198 +18,47 @@ define([
             DiscussionSpecHelper.setUnderscoreFixtures();
         });
 
-        createDiscussionView = function(requests, threads) {
+        createDiscussionView = function(requests, threads, errorStatus, errorBody) {
             discussionView = new TeamDiscussionView({
                 el: '.discussion-module'
             });
             discussionView.render();
-            AjaxHelpers.expectRequest(
-                requests, 'GET',
-                interpolate(
-                    '/courses/%(courseID)s/discussion/forum/%(discussionID)s/inline?page=1&ajax=1',
-                    {
-                        courseID: TeamSpecHelpers.testCourseID,
-                        discussionID: TeamSpecHelpers.testTeamDiscussionID
-                    },
-                    true
-
-                )
-            );
-            AjaxHelpers.respondWithJson(requests, TeamSpecHelpers.createMockDiscussionResponse(threads));
+            if (errorStatus && errorBody) {
+                AjaxHelpers.respondWithError(
+                    requests,
+                    errorStatus,
+                    errorBody
+                );
+            } else {
+                AjaxHelpers.respondWithJson(
+                    requests,
+                    TeamSpecHelpers.createMockDiscussionResponse(threads)
+                );
+            }
             return discussionView;
         };
 
-        createPost = function(requests, view, title, body, threadID) {
```

Comment:
Why remove so many JS tests?

Reply:
These tests are no longer relevant because:
- 4 years ago, we skipped these tests because https://openedx.atlassian.net/browse/TNLARCHIVE-2385
- The various cases I deleted are really inline discussion functionality, rather than related to the teams discussion panel.
- All the tests here are no longer up-to-date. Trying to fix these tests provide little value then, and even less value now.

Reply:
Sounds good to me!

Reply:
Thanks for the context.